common-items.lic

```diff
@@ -268,4 +268,18 @@ module DRCI
       return false
     end
   end
+
+  # There are many variants of a summoned moon weapon (blade, staff, sword, etc)
+  # This function checks if you're holding one then tries to wear it.
+  # https://elanthipedia.play.net/Shape_Moonblade
+  def wear_moon_weapon
+    moon_weapon_regex = /moon/
+    moon_wear_messages = ['telekinetic']
+    if left_hand =~ moon_weapon_regex
+      bput("wear #{left_hand}", *moon_wear_messages)
+    end
+    if right_hand =~ moon_weapon_regex
+      bput("wear #{right_hand}", *moon_wear_messages)
+    end
+  end
```

Comment:
### Why add method to common-items?
This method exists in `common-items` mainly because the two spots in `combat-trainer` that needed to be updated with this logic exists in two different classes. I wasn't sure where to put this method in `combat-trainer` where it'd be accessible by both classes so here instead. It might have use for other scripts, too?

### moon_weapon_regex

There are multiple variants of a summoned moon weapon (moonblade, moonstaff) and many more if you have the [Shape Moonblade](https://elanthipedia.play.net/Shape_Moonblade) spell. The idea here is that if you have a "moon" noun in your hand and you're calling `wear_moon_weapon` then that moon noun is (should be) a summoned moon weapon.

To minimize conflict with already worn moon nouns, like "a trio of moons", then the wear command uses the exact <adjective> <noun> listed in your hand.